### DISCUSSION TITLE
esites Topic 41: How can we address provenience by region rather than continent/country/state (Anthro)?

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1143] [Q&A]
DISCUSSION POINTS: 

For the repatriation website, we would like to be able to include the Saami human remains on the map, but we do not know their country of origin, only  the region (which includes multiple countries).

**Comment by rondlg** [comment]
@magpiedin @peteherbst  

Did we addressed this on the repatriation website already?

**Reply by magpiedin** [reply]
Partly.  
- Currently it's handled via a very rough workaround in the data ([here](https://docs.google.com/document/d/1CAhlaIHdxNk1hihmMqugdLu2JNhkAfo162SK4ACNdmg/edit#heading=h.2wn2z2peqlqp)) -- attributing individuals to each country in the region.  
- Region-polygons aren't on the map / yet.  We would need to find/check an appropriate source with Repat/Anthro, 
  - e.g. https://native-land.ca/resources/api-docs/
